doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+.. _data-ingest:
+
+.. toctree::
+    :hidden:
+
+    conduce-entities
+
+====================
+Intro to Data Ingest
+====================
+
+Visualizing your data in Conduce begins with ingest.  Ingest is the process translating your data into a format that Conduce understands.  Conduce uses entities to define unique objects.  Entities are ingested into datasets.  Datasets hold collections of related entities and provide the interface a user needs to connect their data to a visualization renderer.
+
+Conduce needs a few key pieces of information in order to visualize your data in space and time, those are:
+
+**Identity**
+   Identities distinguish one entity from another.  An entity exists in Conduce as one more more samples.  All samples that describe the same entity share an identity.
+**Location**
+   An entities location describes where in space the entity exists.  Locations are defined in units according to the coordinate system in which they are displayed. (Latitude and Longitude on a world map.)
+**Time**
+   Time defines the moment in time at which the rest of the information describing an entity is valid.
+
+With this information Conduce can build a representation of an object that exists in space and time.  Entity definitions can be more complex, for more information on how to build entities please read :doc:`conduce-entities`.
+
+--------
+Datasets
+--------
+
+Datasets are the containers that hold entities.  A dataset may be thought of as a table.  Each entity in a dataset is similar in nature, but has unique properties.  In order to represent and entity in Conduce, its sample data must be ingested into a dataset.  In order to perform any API operations it is recommended that you generate an API key.  (See :doc:`api-key-creation` for more information.)  The first step in ingesting data is acquiring a dataset ID.  If you are ingesting data into a new dataset you'll need to create the dataset  before entities can be ingested.
```

Comment:
s/In order to represent **and** entity/In order to represent **an** entity

Reply:
"its sample data must be ingested into a dataset"

Should this reference samples explicitly?

Reply:
"dataset  before" - 2 spaces - does 2 spaces matter?

Reply:
The API key sentence should go at the end - it threw me off while going thru the paragraph.  I would add this to the end of the paragraph:

The example below ingests data via the API.  In order to perform API operations ...

Reply:
I'll make it an admonition.

Reply:
Regarding this phrase, "its sample data must be ingested into a dataset," I agonized over that for quite a bit.

If an entity is not defined by a sample, then what is the thing that defines it?  Entity is not a good answer.

Reply:
Well... in our limited API, an entity is defined over all time via an ID, a location, and a set of attributes, or it is defined as a series of samples, each of which contains an ID, a location, a time, and a set of attributes.

The point is that there are 2 distinct ways to define an entity, so the documentation shouldn't give users the impression that there is only one in some places and 2 in other places.